### PR TITLE
feat(web): navigate assets with gestures (next/prev)

### PR DESCRIPTION
**Pull request by kaziu687**
Added ability to navigate using left and right swipe gestures, useful on touchscreen devices

### CODE REVIEW

**Comment by alextran1502** [comment]
Thank you for the PR. As we are trying to merge #10646. This PR will have to wait until that PR get merged to prevent merge conflict

**Review by jrasm91 — APPROVED** [review]
LGTM

**Pull request merged**
